multibase.csv

```diff
@@ -1,26 +1,26 @@
-encoding,          code, description,                                                  status
-identity,          0x00, 8-bit binary (encoder and decoder keeps data unmodified),     default
-base2,             0,    binary (01010101),                                            candidate
-base8,             7,    octal,                                                        draft
-base10,            9,    decimal,                                                      draft
-base16,            f,    hexadecimal,                                                  default
-base16upper,       F,    hexadecimal,                                                  default
-base32hex,         v,    rfc4648 case-insensitive - no padding - highest char,         candidate
-base32hexupper,    V,    rfc4648 case-insensitive - no padding - highest char,         candidate
-base32hexpad,      t,    rfc4648 case-insensitive - with padding,                      candidate
-base32hexpadupper, T,    rfc4648 case-insensitive - with padding,                      candidate
-base32,            b,    rfc4648 case-insensitive - no padding,                        default
-base32upper,       B,    rfc4648 case-insensitive - no padding,                        default
-base32pad,         c,    rfc4648 case-insensitive - with padding,                      candidate
-base32padupper,    C,    rfc4648 case-insensitive - with padding,                      candidate
-base32z,           h,    z-base-32 (used by Tahoe-LAFS),                               draft
-base36,            k,    base36 [0-9a-z] case-insensitive - no padding,                draft
-base36upper,       K,    base36 [0-9a-z] case-insensitive - no padding,                draft
-base58btc,         z,    base58 bitcoin,                                               default
-base58flickr,      Z,    base58 flicker,                                               candidate
-base64,            m,    rfc4648 no padding,                                           default
-base64pad,         M,    rfc4648 with padding - MIME encoding,                         candidate
-base64url,         u,    rfc4648 no padding,                                           default
-base64urlpad,      U,    rfc4648 with padding,                                         default
-proquint,          p,    PRO-QUINT https://arxiv.org/html/0901.4016,                   draft
-base256emoji,      🚀,    base256 with custom alphabet using variable-sized-codepoints, draft
+code,   Unicode,    (UTF-8),    encoding,           description,                                                    status
+NUL,    U+0000,     0x00,       8-bit binary,       (encoder and decoder keeps data unmodified),                    reserved
```

Comment:
Why not keeping the "identity" here? All encodings have a single word name, so it would be consistent.